spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarProducerFactoryTests.java

```diff
@@ -108,6 +112,30 @@ void createProducerWithDefaultTopicAndInterceptor() throws PulsarClientException
 		}
 	}
 
+	@Test
+	@SuppressWarnings("unchecked")
+	void createProducerWithSingleProducerCustomizer() throws PulsarClientException {
```

Comment:
Sorry for the formatting / moving around in the same PR / commit. I will call out what has changed and what is the same to make this easier to review....

**[UNCHANGED-JUST-MOVED-UP]**